src/components/auth/Login.jsx

```diff
@@ -1,12 +1,64 @@
 import { useForm } from 'react-hook-form'
 import { InputField } from './InputField'
+import { useDispatch, useSelector } from 'react-redux'
+import { startLogin } from '../../stores/userThunks'
+import { Alert } from '../alert'
+import { useEffect, useMemo, useState } from 'react'
 
 export const Login = () => {
    const { control, handleSubmit } = useForm()
+   const dispatch= useDispatch();
+   const {message, status} = useSelector(state => state.user)
+
+   const [alertMessage, setAlertMessage] = useState({
+      message:'',
+      backgroundColor: '',
+   }); 
 
    const onSubmit = (data) => {
-      console.log(data)
+      dispatch(startLogin(data))
    }
+   const isAuthenticating = useMemo(() => status === 'checking', [status])
+
+   const handlealertMessage = (alertMessage) => {
+      switch (alertMessage) {
```

Comment:
Instead of using a switch statement please use a dictionary, this is a better practice.